styles/style.css

```diff
@@ -66,7 +71,8 @@ p {
 }
 
 .navigation__item {
-  padding: 0.5rem 0;
+  padding: 0.5rem 0.8rem;
+  flex-grow: 1;
```

Comment:
Can you make the navigation links change on hover?

Reply:
A hover was created for my navigation links